src/pipemode_op/Dataset/src/pipemode_dataset_op_kernel.cpp

```diff
@@ -194,11 +210,12 @@ class PipeModeDatasetOp : public DatasetOpKernel {
             ~Iterator() {
                 if (benchmark_) {
                     int64_t read_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(read_time_).count();
-                    std::cout << "PipeModeDatasetOp::Dataset::Iterator read_time_ms: " << read_time_ms << std::endl;
-                    std::cout << "PipeModeDatasetOp::Dataset::Iterator read_bytes: " << read_bytes_  << std::endl;
+                    std::cout << "PipeModeDatasetOp::Dataset::Iterator epoch read_time_ms: " << read_time_ms
```

Comment:
Apologies for not picking this up in the first review. This needn't be an epoch. 

1) The algorithm that's acting on the data may just be a single pass, in which case there's no notion of an epoch.
2) The Dataset may be discarded before the entire fifo is drained.

I would replace epoch with 'total'.

Reply:
fixed in the new revision.